mods/wool/init.lua

```diff
@@ -26,6 +26,12 @@ for i = 1, #dyes do
 	}
 end
 
+-- Alternative recipe for strings.
+minetest.register_craft({
+		output="farming:string 4",
```

Comment:
You're using two tabs here when you should use just one. Overall, you should try to "blend in" formatting-wise, for example adding spaces around `=`.

Reply:
> You're using two tabs here when you should use just one. Overall, you should try to "blend in" formatting-wise, for example adding spaces around `=`.

I changed the code to fit the style. Thank you!